### PR TITLE
bz1656397 amended backup and recovery steps for clarity

### PR DESCRIPTION
**Pull request by sheriff-rh**
Bug 1656397:
https://bugzilla.redhat.com/show_bug.cgi?id=1656397

@jiajliu  please take a look. 

@vikram-redhat  FYI

### CODE REVIEW

File: day_two_guide/topics/proc_backing-up-master.adoc [hunk 1]
```diff
@@ -40,6 +40,8 @@ Because the masters are also nodes, back up the entire `/etc/origin` directory.
 You must perform the following steps on each master node.
 ====
 
+. Create a backup of the pod definitions, located xref:../day_two_guide/topics/proc_backing-up-etcd.adoc[here].
```
Comment (kalexand-rh) [inline day_two_guide/topics/proc_backing-up-master.adoc]: no xrefs in modules

Comment (vikram-redhat) [reply day_two_guide/topics/proc_backing-up-master.adoc]: @kalexand-rh it is for 3.11 so it should be ok to have xrefs in modules.

Comment (vikram-redhat) [reply day_two_guide/topics/proc_backing-up-master.adoc]: @sheriff-rh I believe the issue is that you are linking to a module, and not to the assembly. The module doesn't exist by itself so the build is failing. 

You probably need to replace this with:

```suggestion
. Create a backup of the pod definitions, located xref:../day_two_guide/environment_backup.adoc#backing-up-etcd_environment-backup[here].
```
Trying the new suggested feature to see if you can simply accept the change without needing to re-push. :)

**Comment by sheriff-rh** [comment]
That corrected it, thanks @vikram-redhat !

**Comment by sheriff-rh** [comment]
Changes have been verified in bugzilla; merging.

**Pull request merged**